test/morphology/test_black_hat.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import torch
+import kornia.morphology as m
+import kornia.testing as utils  # test utils
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+class TestBlackHat(utils.BaseTester):
+
+    def test_smoke(self, device, dtype):
+        kernel = torch.rand(3, 3, device=device, dtype=dtype)
+        assert m.se_to_mask(kernel) is not None
+
+    def test_batch(self, device, dtype):
+        input = torch.rand(3, 2, 6, 10, device=device, dtype=dtype)
+        kernel = torch.rand(3, 3, device=device, dtype=dtype)
+        test = m.black_hat(input, kernel)
+        assert input.shape == test.shape == (3, 2, 6, 10)
+
+    def test_value(self, device, dtype):
+        input = torch.tensor([[0.5, 1., 0.3], [0.7, 0.3, 0.8], [0.4, 0.9, 0.2]],
+                             device=device, dtype=dtype).unsqueeze(0).unsqueeze(0)
```

Comment:
can you try better formatting ? to make more readable the tests? use `[]` instead of unsqueeze, it's more pythonic